tests/aksservice_test.go

```diff
@@ -0,0 +1,42 @@
+package aks_test
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/api/core/v1"
+)
+
+var _ = Describe("AKS-Service Tests : ", func() {
+	//Adding delay just to let flux deploy the manifests.
+	time.Sleep(30 * time.Second)
+
+		Context("Traefik service is running", func() {
+			service := getService(clientset,"traefik", "admin")
+			It("should have loadbalancer ip set", func() {
+				Expect(service.Status.LoadBalancer.Ingress[0].IP).ShouldNot(BeNil())
```

Comment:
is it worth checking it's an ip address?

Reply:
Test that it fails when loadbalancer ip is not set